src/Components/Input/Input.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { render } from "@testing-library/react-native";
```

Comment:
There is a changeText event firing function in [the lib](https://www.native-testing-library.com/docs/api-events)